src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -317,18 +315,20 @@ public void noPropertiesWarnings() throws Exception {
         r.waitUntilNoActivity();
         WorkflowRun b1 = p.getLastBuild();
         assertEquals(1, b1.getNumber());
-        r.assertLogNotContains("WARNING: The 'properties' step will remove all 'JobProperty's currently configured in this job, either from the UI or from an earlier 'properties' step.", b1);
+        r.assertLogNotContains(Messages.JobPropertyStep__could_remove_warning(), b1);
 
         // Now verify that we don't get any messages about removing properties when a property actually gets removed as
         // we add a new one.
         sampleRepo.write("Jenkinsfile", "echo \"branch=${env.BRANCH_NAME}\"\n"
-                + "properties([[$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', numToKeepStr: '1']]])");
-        sampleRepo.git("add", "Jenkinsfile");
+                + (HAVE_SYMBOL ?
+                "properties([buildDiscarder(logRotator(numToKeepStr: '1'))])" :
+                "properties([[$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', numToKeepStr: '1']]])"));
+                sampleRepo.git("add", "Jenkinsfile");
         sampleRepo.git("commit", "--all", "--message=flow");
 
         WorkflowRun b2 = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
-        r.assertLogNotContains("WARNING: The 'properties' step will remove all 'JobProperty's currently configured in this job, either from the UI or from an earlier 'properties' step.", b2);
-        r.assertLogNotContains("WARNING: Removing existing job property", b2);
+        r.assertLogNotContains(Messages.JobPropertyStep__could_remove_warning(), b2);
+        r.assertLogNotContains(Messages.JobPropertyStep__removed_property_warning("disableConcurrentBuilds"), b2);
```

Comment:
Could probably use `TheClass.class.getAnnotation(Symbol.class).getValue()` to avoid hardcoding this too, though it is not so important because symbols are stable.